### PR TITLE
[14.0][ADD] product_multi_barcode_menu

### PR DESCRIPTION
**Pull request by ilyasProgrammer**
Adds menu entry in inventory configuration to see all barcodes.

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
LGTM! 

@rousseldenis DILGTY? 😀

File: product_multi_barcode_menu/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Multiple barcodes menu",
```
Comment (rousseldenis) [inline product_multi_barcode_menu/__manifest__.py]: IMHO product_multi_barcode_stock_menu should be better

Comment (ilyasProgrammer) [reply product_multi_barcode_menu/__manifest__.py]: ok

**Review by SicurSam — APPROVED** [review]
LGTM

**Review by rousseldenis — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@rousseldenis can we merge too?

**Comment by francesco-ooops** [comment]
@OCA/logistics-maintainers can we merge?

**Pull request merged**